doc/source/myrocks/zenfs.rst

```diff
@@ -0,0 +1,99 @@
+.. _zenfs:
+
+=========================================
+RocksDB with ZenFS
+=========================================
+
+Implemented in Percona Server for MySQL 8.0.26-16.
+
+`ZenFS <https://zonedstorage.io/projects/zenfs/>`__ is a file system plugin which places files into zones on a raw zoned block device (ZBD) using the RocksDB File System interface. ZenFS depends on ``libzbd`` and requires a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel must be configured with `zone block device support enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__. 
+
+.. note::
+
+    The Linux kernel 5.9 and later support NVMe Zoned Namespaces. The Linux distributions that currently have ``ZNS support`` are listed `here <https://zonedstorage.io/distributions/linux/>`__. 
+
```

Comment:
See my comment above, there are currently no suitable kernels available yet that can run MyRocks ZenFS out of the box correctly.  We can perhaps add these back once suitable kernels and distributions become available maybe in 2022.  For now, there is no existing distribution and kernel combination that will work out of the box.

Reply:
OK. I'll remove the note.

Reply:
I'm sorry, but I disagree about removing this. We need to provide proper guidance and set expectations to users that intend to set up PS on ZenFS using our documentation. We need to make clear that our documentation expects an OS environment that hes been setup correctly and the installation and configuration is out of the scope of this document.

Therefore, we should point to the canonical resource that tracks distribution support and could potentially contain additional resources that help users to get started with preparing the OS environment. They may be inaccurate/incomplete right now, but that won't change if they aren't known/used and are being established as the recommended instructions.